config/sync/field.field.node.stanford_news.su_news_tags.yml

```diff
@@ -0,0 +1,29 @@
+uuid: cf2ecdeb-422a-4201-8a45-c6dbf5bff39c
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_news_tags
+    - node.type.stanford_news
+    - taxonomy.vocabulary.su_news_tag_filters
+id: node.stanford_news.su_news_tags
+field_name: su_news_tags
+entity_type: node
+bundle: stanford_news
+label: "News Tags/Filters\t"
+description: 'These will be used as filters for the future Filtered Lists Paragraph for News. They do not display on the News item or News Teaser.'
```

Comment:
"Tag this news item with categories for use with the Filtered List Paragraph. Tags do not display on the News page or News Teasers. Learn more (and then link to https://sitesuserguide.stanford.edu/build/paragraphs/filtered-lists-paragraph.)